circleci.tf

```diff
@@ -36,14 +36,44 @@ variable "services_instance_type" {
 
 variable "builder_instance_type" {
     description = "instance type for the builder machines.  We recommend a r3 instance"
-    default = "r3.2xlarge"
+    default = "r3.4xlarge"
```

Comment:
Matches the default at https://github.com/circleci/enterprise-setup/blob/3a3a3a3/terraform.tfvars#L9, let me know if it should be vice-versa.